src/users/CreatesUsers.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const async = require('async');
+const LoginsUsers = require('./LoginsUsers');
+const DB = require('../db/db');
+const config = require('../../config');
+
+// callback(err, docId, docBody)
+const createUserDoc = (id, password, callback) => {
+  new LoginsUsers().hashPassword(password, (err, hash) => {
+    if (err)
+      return callback(err);
+
+    callback(null, {id, hash});
+  });
+};
+
+// callback(err, docId2docBodyMap)
+const createAliasesDocs = (id, aliases, creationDate, callback) => {
+  const docs = {};
+
+  aliases.forEach(alias => {
+    const docId = `alias:${alias.type}:${alias.value}`;
+    const docBody = {
+      id,
+      date: creationDate,
+      public: alias.public === true
+    };
+
+    docs[docId] = docBody;
+  });
+
+  setImmediate(callback, null, docs);
+};
+
+class CreatesUsers {
```

Comment:
@elmigranto Out of curiosity, is this some ES6 standard naming convention for classes? 

(we generally use nouns as class names, it's the first time I see verbs conjugated at the 3rd person... No biggy though).